scripts/system/effect.js

```diff
@@ -25,7 +25,15 @@ export default class AgeOfSigmarEffect extends ActiveEffect {
     }
 
     fillDerivedData(actor, change) {
-        change.value = eval(Roll.replaceFormulaData(change.value, actor.getRollData()))
+        let data = eval(Roll.replaceFormulaData(change.value, actor.getRollData()))
+        //Foundry Expects to find a String anothing but Strings 
+        //Raw Numbers don't work anymore
+        if(Number.isInteger(data)) {
+            change.value = "" + data;
+        } else {
+            change.value = data;
+        }
```

Comment:
This could be cleaner with simply `change.value = change.value.toString()`

Reply:
making everything a string may defeat the purpose of this function. The core problem why numeric derived data doesn't work is because foundry runs `const n = Number.fromString(value);` this results in NaN. The follow check for NaN then causes no value to be applied `if ( !isNaN(n) ) update = current + n;` . If we ever needed more complecated data like an object for a costum effect we would need to reparse it for later use. The problem we have is numbers because foundry wants numbers to be strings

Reply:
Fair points, but I think I simply had in mind this, which is more selective.

```js
if (Number.isInteger(data))
{
     change.value = data.toString()
}
```

Additionally, you may want to replace the integer check, as floating points *could* come up (mainly effects that might have anything, which would multiply by a value of `0.5`, I've seen them come up unexpectedly in Wrath & Glory).

perhaps `typeof === "number"`?

Reply:
`typeof data === "number"` should work